man/varlinkctl.xml

```diff
@@ -120,11 +120,12 @@
       </varlistentry>
 
       <varlistentry>
-        <term><command>introspect</command> <replaceable>ADDRESS</replaceable> <replaceable>INTERFACE</replaceable></term>
+        <term><command>introspect</command> <replaceable>ADDRESS</replaceable> [<replaceable>INTERFACE…</replaceable>]</term>
 
-        <listitem><para>Show interface definition of the specified interface provided by the specified
-        service. Expects a service address in one of the formats described above and a Varlink interface
-        name.</para>
+        <listitem><para>Show the interface definitions of the specified interfaces provided by the specified
+        service. Expects a service address in one of the formats described above and optionally one or more
+        Varlink interface names. If no interface names are specified shows all provided interfaces by the
```

Comment:
I'd like to insert `,` between `specified` and `shows`:
`If no interface names are specified, shows all provided interfaces...`